src/style.css

```diff
@@ -2,0 +1,51 @@
+@font-face {
+  src: url(../src/fonts/Avenir.ttc);
```

Comment:
Ensure that the font file path is correct. The path `../src/fonts/Avenir.ttc` should be verified to ensure the font loads correctly.